src/quo2/components/wallet/wallet_overview/style.cljs

```diff
@@ -20,8 +24,10 @@
    :height        32})
 
 (defn color-metrics
-  [metrics]
-  (if (= metrics :positive) colors/success-50 colors/danger-50))
+  [metrics theme]
+  (if (= metrics :positive)
+    colors/success-50
```

Comment:
Probably success needs to be themed too?

Reply:
oh you're right, I didn't notice that! I'll fix that...

Reply:
fixed/done